tests/foreman/ui_airgun/test_rhai.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import upload_manifest as up_man
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME, DISTRO_RHEL7
+from robottelo.vm import VirtualMachine
+
+
+pytestmark = pytest.mark.usefixtures("attach_subscription")
+
+
+@pytest.fixture(scope="module")
+def module_org():
+    org = entities.Organization(name="insights_{0}".format(
+        gen_string("alpha", 6))).create()
+    with manifests.clone() as manifest:
+        up_man(org.id, manifest.content)
+    yield org
+    org.delete()
+
+
+@pytest.fixture(scope="module")
+def activation_key(module_org):
+    ak = entities.ActivationKey(
+        auto_attach=True,
+        content_view=module_org.default_content_view.id,
+        environment=module_org.library.id,
+        name=gen_string("alpha"),
+        organization=module_org
+    ).create()
+    yield ak
+    ak.delete()
+
+
+@pytest.fixture(scope="module")
+def attach_subscription(module_org, activation_key):
+    for subs in entities.Subscription(organization=module_org).search():
+        if subs.read_json()["product_name"] == DEFAULT_SUBSCRIPTION_NAME:
+            # "quantity" must be 1, not subscription["quantity"]. Greater
+            # values produce this error: "RuntimeError: Error: Only pools
+            # with multi-entitlement product subscriptions can be added to
+            # the activation key with a quantity greater than one."
+            activation_key.add_subscriptions(data={
+                "quantity": 1,
+                "subscription_id": subs.id,
+            })
+            break
```

Comment:
i think some `else:` statement for `for` loop would be useful to catch the case when subscription wasn't found at all - with current implementation it will just silently do nothing and pass. Thoughts?

Reply:
Done